etc/kernel-launchers/kubernetes/scripts/launch_custom_resource.py

```diff
@@ -47,15 +47,15 @@ def launch_custom_resource_kernel(kernel_id, port_range, response_addr, public_k
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
-    parser.add_argument('--RemoteProcessProxy.kernel-id', dest='kernel_id', nargs='?',
+    parser.add_argument('--kernel-id', dest='kernel_id', nargs='?',
```

Comment:
If you're going to use the truncated options for this application, then you need to update https://github.com/jupyter/enterprise_gateway/blob/master/etc/kernelspecs/spark_python_operator/kernel.json to use the truncated forms as well.

Reply:
thanks @kevin-bates for catching this. Better will be to add an alias rather than removing it from the launch script so that deployment files with old arguments will also work.

Reply:
Sounds good @amarinder-bindra.  I think EG 3.0 would be a good time to move to the truncated forms.